src/metabase/api/collection.clj

```diff
@@ -321,12 +322,12 @@
     always-true-hsql-expr))
 
 (defmulti ^:private post-process-collection-children
-  {:arglists '([model collection rows])}
-  (fn [model _ _]
+  {:arglists '([model options collection rows])}
+  (fn [model _ _ _]
```

Comment:
passing options seems like it should come last, is there a good reason to put it second? (basically in the 1st position, really)

Reply:
I think there are two reasons here:
- one, maybe `options` is a bad name. It's the whole `options` bundle that's sent to the collection items API, so it's less "some optional bits that tell this function how to behave" and more "the context in which this function is being run", if that makes sense?
- two, silly, but it's nice to be able to do `(->> rows (process-a) (process-b) (post-process-collection-children))`

Reply:
Cool, makes sense. Might be good to change the name then. `context` is a similarly bad name but at least it doesn't make that part feel "optional". Up to you.